puppet/zulip/files/cron.d/catch-up-soft-deactivated-users

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Cron job to catch-up soft deactivated users daily.
+0 17 * * * zulip /home/zulip/deployments/current/manage.py soft_deactivate_users -c >/dev/null
```

Comment:
At the very least, I think this should be the same cron file used to soft-deactivate users (and just an additional line in it), but see my comment above that it might be even better to just have this be a part of what the main soft-deactivation cron job does.

Reply:
Yep, having it in the same file is nicer! But, I wasn't sure if that was the way to go. I'll change this.